src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/BlacklistedUPRNBean.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.ons.ctp.integration.contactcentresvc;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ctp.common.domain.UniquePropertyReferenceNumber;
+import uk.gov.ons.ctp.common.event.EventPublisher.Channel;
+import uk.gov.ons.ctp.integration.contactcentresvc.config.AppConfig;
+
+/** This class contains UPRNs which have been blacklisted. */
+@Component
+public class BlacklistedUPRNBean {
```

Comment:
Not a big point, but in an ideal world we would have a unit test class for this with full code coverage.  I know we haven't got that for the CCS Postcode bean, but ideally it would have that too.

Reply:
I'll add if there is time.